src/HydraAuctionOnchain/Helpers.hs

```diff
@@ -127,7 +181,48 @@ ponlyOneInputFromAddress = phoistAcyclic $
 pserialise :: PIsData a => Term s (a :--> PByteString)
 pserialise = phoistAcyclic $ plam $ \x -> pserialiseData #$ pforgetData $ pdata x
 
+ptxOutContainsAuctionEscrowToken :: Term s (PCurrencySymbol :--> PTxOut :--> PBool)
+ptxOutContainsAuctionEscrowToken = phoistAcyclic $
+  plam $ \auctionCs txOut ->
+    (pvalueOf # (pfield @"value" # txOut) # auctionCs # auctionEscrowTokenName)
+      #== 1
+
+ptxOutContainsStandingBidToken :: Term s (PCurrencySymbol :--> PTxOut :--> PBool)
+ptxOutContainsStandingBidToken = phoistAcyclic $
+  plam $ \auctionCs txOut ->
+    (pvalueOf # (pfield @"value" # txOut) # auctionCs # standingBidTokenName)
+      #== 1
```

Comment:
It's not a big deal, but these functions seem out-of-place in this module, which otherwise has general functions that know nothing about auctions.

For example, in my implementation, I put everything that has to do with the auction token names into its own module:
https://github.com/mlabs-haskell/hydra-auction/blob/9801baa2d2ced80fefbbaab526fdd7cb0cd6aa22/hydra-auction-onchain/src/HydraAuction/Onchain/Types/Tokens.hs

Reply:
Generally, I'm wary of various "Helper" or "Utils" modules becoming too large and diverse. It becomes mentally taxing to read them, later on.

However, without the auction-specific function (and other arbitrary functions you may be tempted to throw here), there's a clear theme to the contents of this module -- enhancing Plutus API to provide more useful predicates/extractors from TxInfo and its constituents.

Functions from this module could eventually be contributed upstream, if people like them.